config/llvm_header.inc

```diff
@@ -245,6 +245,15 @@ define void @take_search_step(ptr %subject) {
   ret void
 }
 
+define i256 @hook_BYTES_length256(ptr %b) {
+entry:
+  %hdr = load i64, ptr %b
+  %len = and i64 %hdr, 1099511627775 ; @LENGTH_MASK@
```

Comment:
This is not ideal. Since LENGTH_MASK is a configurable value, having it hardcoded here can potentially create confusion. Maybe you can write a C++ function that is just wrapper for the `len` macro and use it here.

There are more instances of the value being hard coded. We will need to apply this to all of them.